node/clients/index.ts

```diff
@@ -0,0 +1,45 @@
+import { ClientsConfig, IOClients } from '@vtex/api'
+
+import Canonicals from './canonicals'
+import Robots from './robots'
+import Routes from './routes'
+import SiteMap from './sitemap'
+
+const TEN_SECONDS_MS = 10 * 1000
+const THREE_SECONDS_MS = 3 * 1000
+
+export class Clients extends IOClients {
+  public get canonicals() {
+    return this.getOrSet('canonicals', Canonicals)
+  }
+
+  public get robots() {
+    return this.getOrSet('robots', Robots)
+  }
+
+  public get routes() {
+    return this.getOrSet('routes', Routes)
+  }
+
+  public get sitemap() {
+    return this.getOrSet('sitemap', SiteMap)
+  }
+}
+
+export const clients: ClientsConfig<Clients> = {
```

Comment:
why not moving this to index.ts so we have only one single file to look for configurations